### DISCUSSION TITLE
Different method of blocking ads

### DISCUSSION THREAD

**Discussion by CharlieS1103** [AnanthVivekanand/spotify-adblock#57] [General]
Hello, using Spicetify I can block audio ads without ever touching anything network-related, using built-in Spotify methods.
Of course, removing the UI ads can be done simply with javascript in the extension.
In the case that you would like to work on either the Spicetify version with me(rather simple, would just be a means of deleting the HTML components for the UI ads, and the audio ads blocking is more or less just three lines of code which I've already written), or would like information on how to disable audio ads without the proxy and such, just reply.

**Comment by AnanthVivekanand** [comment]
Sounds very interesting! Unfortunately I just don't have time right now to contribute due to personal reasons but I'd be excited to see your work when you release it. I'd be happy to add a link to your project in my README, but I think I'll keep maintaining the proxy-based approach in this repo just for users who don't want to use Spicetify etc. 

If I find time that I can set aside to contribute, I'd be happy to help in any way I can. I'll let you know if that's ever the case. Truly thanks for offering though.
